package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "backport-action",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "private": true,
   "description": "GitHub action to automatically backport pull requests",
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "format": "prettier --write **/*.ts",
-    "format-check": "prettier --check **/*.ts",
+    "format": "prettier --write src/",
+    "format-check": "prettier --check src/",
     "package": "ncc build --source-map --license licenses.txt",
-    "all": "npm run build && npm run format && npm run package"
+    "test": "jest --silent",
+    "all": "npm run format && npm run build && npm run package && npm test"
```

Comment:
Do we want to have the quiet command here?

Reply:
I think I prefer quiet for locally running this. If tests fail those failure are still reported. The only thing missing are our own console logs. For CI build I agree that we should run it more explicitly as non-quiet.